Khan/perseus
# Stop keypad button clicks from removing focus from the input

## Summary:

Currently in the math input component, if a user has focus on the input field and then clicks
a keypad button, it shifts focus to the button before re-focusing on the input field. This
creates a jarring effect with the focus outline, and it causes the screen reader to read out the
full input value on every single click without reading the button key at all. I think it also sometimes
causes the screen reader to falsely start reading "popover closed" before getting interrupted.

To fix this, I'm adding `e.preventDefault()` on mouse down to the keypad buttons. This will
stop them from stealing focus from the input.

In this PR:
- Updated WB Clickable version in Perseus to support onMouseDown
- Added `e.preventDefault()` to the keypad buttons to stop them from stealing focus from the input

Issue: https://khanacademy.atlassian.net/browse/LC-1370

## Test plan:

Storybook
- Go to http://localhost:6006/iframe.html?id=perseus-components-math-input--default-with-basic-button-set&viewMode=story
- Open the math keypad
- Click on the keypad buttons with the cursor
- Confirm that focus didn't shift back and forth between the button and the input field
- Confirm that the button that was pressed is read out by the screen reader

### Before

Demo:
1. using keyboard to type
2. then using mouse to click keypad buttons

https://github.com/Khan/perseus/assets/13231763/c3313e5a-08fc-45dc-b942-2cd63f5a7b48

### After

Demo:
1. using keyboard to type
2. then using mouse to click keypad buttons

https://github.com/Khan/perseus/assets/13231763/4d918692-ebaa-4ca1-8ea0-231f17539f02
